content/pages/constitution.md

```diff
@@ -211,19 +211,44 @@ Committee office, M3 1003.
     2. The current committee will then select evaluate the applications
        received and select applicants for interviews;
 
-    3. The current committee will select interviewers. Interviewers should be
-       chosen by the current committee, and are expected to disclose any
+    3. A single interview shall be held for each candidate. Interviews shall
+       not exceed 30 minutes in length;
+
+    4. The current committee will select interviewers. Two interviewers should
+       be chosen by the current committee, and are expected to disclose any
        conflicts of interest when presented with the list of candidates. If a
        conflict of interest arises, an alternate interviewer shall be chosen.
        In any case, if no interviewer is explicitly chosen and there is no
-       conflict of interest, the current Chair shall conduct the interviews;
+       conflict of interest, the current Chair and Secretary shall conduct the
+       interviews;
+
+    5. Two interviewers must be present at every interview, and each shall
```

Comment:
++ should
I think that everyone on the committee has an understanding of the core values of wics and while it's definitely beneficial to have two people, it's not always necessary. I think that as long as the interviewer is able to write down the key notes (and we're asking the right questions), the committee will have a pretty good idea if the interviewee is a good fit. Also, sometimes scheduling conflicts make it very difficult to have more than one person interviewing. :)